io.embrace.sdk/Scripts/IEmbraceProvider.cs

```diff
@@ -71,7 +67,18 @@ bool RecordCompletedSpan(string spanName, long startTimeMs, long endTimeMs, int?
         void setShakeListener(UnityShakeListener listener);
         #endif
         #endif
+        
+        [Obsolete("SetUserPersona is deprecated and will be removed from a future release. Use AddUserPersona instead", false)]
+        void SetUserPersona(string persona);
```

Comment:
If this ends up being a major version bump then IMO we can just delete all these

Reply:
I would love to.